### DISCUSSION TITLE
未来更新功能疑问

### DISCUSSION THREAD

**Discussion by lingheaila** [mikumifa/BiliShareMall#4] [Q&A] [answered]
1.价格升降序加入查询条件 
2.不选择类型查询 
3.关注特定产品历史最低价并发送邮件提醒
这三个功能之后有机会给加上吗？

**Comment by mikumifa** [comment]
不选择类型查询是啥意思，是爬取的时候不指定类型吗

**Reply by lingheaila** [reply]
对

**Reply by mikumifa** [reply]
这个实际上用的是b站的接口去爬取的，b站必须要指定类型。所以没办法。@lingheaila

**Reply by mikumifa** [reply]
价格降序可以搞定

**Reply by lingheaila** [reply]
可以的，我用大佬你之前的那个py脚本就可以，直接注释参数就行。还是说用了其他接口？
![image](https://github.com/user-attachments/assets/d07bb42d-e3d5-48d2-af2f-bb5b52d593ff)

**Reply by lingheaila** [reply]
阿b自己集市的默认参数也是不带的

![Uploading image.png…]()

**Reply by mikumifa** [reply, marked as answer]
okk，我后面加上

**Reply by lingheaila** [reply]
好的捏，感谢

**Comment by mikumifa** [comment]
第三点应该做不到
